aws_lambda_powertools/utilities/batch/middlewares.py

```diff
@@ -54,3 +56,58 @@ def batch_processor(
         processor.process()
 
     return handler(event, context)
+
```

Comment:
I can't seem to comment on the other `batch_processor` middleware... **suggestion** Bring the `batch_processor` under `base.py`